langlib/lang.runtime/src/main/ballerina/runtime.bal

```diff
@@ -84,10 +84,15 @@ public isolated function getStackTrace() returns StackFrame[] {
     int i = 0;
     CallStackElement[] callStackElements = externGetStackTrace();
     lang_array:forEach(callStackElements, function (CallStackElement callStackElement) {
-                                stackFrame[i] = new java:StackFrameImpl(callStackElement.callableName,
-                                callStackElement.moduleName, callStackElement.fileName, callStackElement.lineNumber);
-                                i += 1;
-                            });
+        if (callStackElement["moduleName"] is string) {
```

Comment:
Why aren't we using field access here?

Reply:
since the `moduleName` is `string` in this case, we can use the field access directly. (Anyway this part has been removed with the new changes addressed with the commit https://github.com/ballerina-platform/ballerina-lang/pull/30998/commits/e88161f75a7c26f69317defdd98388059a53ab5d)